parts_list/master_parts_list_raw.csv

```diff
@@ -1,144 +1,145 @@
-Part Name,Project Ref Code,Model/Config,Site,Link,Sold in Packs of,Total Used in Project,Quantity to Buy,Price Each,Price Total,Used in Sections

```

Comment:
Seems like this file got messed up again. 
Protip: in the future (if you haven't already) create a separate commit when you make changes to this file so that you can easily revert that one commit without having to revert all :)

Reply:
I am at a loss to understand what is happening here. I used Excel and saved the file back as a .csv and it is showing everything as changed. (This is something I do all the time with other files.)

Reply:
@dcschooley You are correct. The file has Windows line endings after begin exported. If you are unable to export with Unix line endings I would suggest re-saving the file in a editor configured to use Unix line endings.

Reply:
It's an easy fix.

Reply:
> Protip: in the future (if you haven't already) create a separate commit when you make changes to this file so that you can easily revert that one commit without having to revert all :)

That sounds like a good way. I've fixed the line endings.